src/status_im/contexts/onboarding/create_password/view.cljs

```diff
@@ -112,7 +113,7 @@
                           #(utils.string/at-least-n-chars? % 10))]
     (->> password
          (validations)
-         (zipmap [:lower-case? :upper-case? :numbers? :symbols? :long-enough?]))))
+         (zipmap (conj constant/password-tips :long-enough?)))))
```

Comment:
Should `:long-enough?` also be included in the constant password-tips?

Reply:
don't think so, since it's not a "tip" but a validation rule. It doesn't really belong here, but didn't want to add more changes than needed